packages/task/src/node/task-problem-collector.ts

```diff
@@ -44,4 +44,24 @@ export class ProblemCollector {
         });
         return markers;
     }
+
+    processLineMatchBegin(line: string): boolean {
+        let match = false;
+        this.lineMatchers.forEach(lineMatcher => {
```

Comment:
using `Array.some()` could be more efficient and readable. same comment applies to line 60

Reply:
fixed, done !